ucfcs/GrowthPlus
# Julio/flashcard fragments

This PR implements the necessary fragments and adds a category field to lesson schema. 

To test: 
Uninstall , rebuild and run the app. 
Create a child and save the realm file, make sure the category field was added to the flashcard schema.
